src/Pact/Native/Db.hs

```diff
@@ -307,10 +307,11 @@ guardTable i t = evalError' i $ "Internal error: guardTable called with non-tabl
 
 enforceBlessedHashes :: FunApp -> ModuleName -> Hash -> Eval e ()
 enforceBlessedHashes i mn h = do
-  mm <- HM.lookup mn <$> view (eeRefStore.rsModules)
+  mmRs <- fmap fst . HM.lookup mn <$> view (eeRefStore.rsModules)
+  mm <- maybe (HM.lookup mn <$> use (evalRefs.rsLoadedModules)) (return.Just) mmRs
```

Comment:
I propose this as easier to read:

```
mmRs <- fmap fst . HM.lookup mn <$> view (eeRefStore.rsModules)
mmRefs <- HM.lookup mn <$> use (evalRefs.rsLoadedModules)
case mmRs <|> mmRefs of
```

Reply:
If I'm not mistaken, this would result in a needless state lookup even if `mmRs` is found, which is a concern as this impacts database access performance: we're in a strict state monad so I'm assuming `mmRefs` would be evaluated.